src/history/hs.c

```diff
@@ -460,14 +460,15 @@ __hs_insert_record(WT_SESSION_IMPL *session, WT_CURSOR *cursor, const uint32_t b
     WT_RET(cursor->update(cursor));
 
     /* Append a delete record to represent stop time pair for the above insert record */
-    cursor->set_key(
-      cursor, btree_id, key, upd->start_ts, upd->txnid, stop_ts_pair.timestamp, stop_ts_pair.txnid);
+    // cursor->set_key(
```

Comment:
Nice. When #5180 goes in, you should be able to remove this.

Reply:
@andym03 #5180 has been merged, are we able to remove these commented lines now